src/models/incident/saga.test.js

```diff
@@ -135,11 +135,33 @@ describe('models/incident/saga', () => {
         payload,
       };
       const error = new Error('Whoops!!1!');
+      const detail = 'Some error message';
+      error.response = {
+        status: 400,
+        jsonBody: {
+          detail,
+        },
+      };
 
-      return expectSaga(patchIncident, action)
+      jest.spyOn(global, 'alert');
+
+      await expectSaga(patchIncident, action)
         .provide([[matchers.call.fn(authPatchCall), throwError(error)]])
         .put(patchIncidentError({ type, error }))
         .silentRun();
+
+      expect(global.alert).not.toHaveBeenCalled();
+
+      error.response.status = 403;
+
+      await expectSaga(patchIncident, action)
```

Comment:
Can't we inspect the return values here instead of spying on a global function that we added just for this purpose?

Reply:
But the return value doesn't contain anything I need. I just need to know that the alert method has been called.